.github/workflows/validation.yml

```diff
@@ -0,0 +1,38 @@
+name: Validate Modified Chains
+
+on:
+  workflow_run:
+    workflows: ["Maintainer Approval"]
+    types:
+      - completed
+
+jobs:
+  validate-modified:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.workflow_run.head_commit.id }}
+        fetch-depth: 0  # Ensure the full history is fetched
+
+    - name: Fetch base branch
+      run: |
+        git fetch origin ${{ github.event.workflow_run.base_sha }}
+
+    - name: "Install just"
```

Comment:
We could do this in a followup PR, but this would be neater https://github.com/extractions/setup-just